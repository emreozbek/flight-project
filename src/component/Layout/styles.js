export const styles = theme => ({
  background: {
    display: "block",
    content: "",
    width: "100%",
    height: theme.spacing.unit * 30,
    backgroundColor: theme.palette.secondary.main,
    position: "absolute",
    zIndex: -1
  },
  wrapper: {
    padding: `${theme.spacing.unit * 8}px ${theme.spacing.unit * 2}px 0 ${theme
      .spacing.unit * 2}px`,
    maxWidth: 720 + theme.spacing.unit * 4,
    marginLeft: "auto",
    marginRight: "auto"
  }
});
