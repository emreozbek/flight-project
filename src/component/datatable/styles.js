export const styles = theme => ({
  table: {
    minWidth: 720
  },
  tableWrapper: {
    overflowX: "auto"
  },
  searchBox: {
    paddingTop: theme.spacing.unit * 2,
    paddingLeft: theme.spacing.unit * 2,
    paddingRight: theme.spacing.unit * 2,
    paddingBottom: theme.spacing.unit * 2
  }
});
