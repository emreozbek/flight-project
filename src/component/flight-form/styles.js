export const styles = theme => ({
  padding: {
    padding: theme.spacing.unit * 2
  },
  marginLeft: {
    marginLeft: theme.spacing.unit * 2
  }
});
