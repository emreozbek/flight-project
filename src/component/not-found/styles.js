export const styles = theme => ({
  padding: {
    padding: theme.spacing.unit * 2
  }
});
