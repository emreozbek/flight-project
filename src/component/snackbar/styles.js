export const styles = theme => ({
  close: {
    padding: theme.spacing.unit / 2
  },
  error: {
    backgroundColor: theme.palette.error.main
  }
});
