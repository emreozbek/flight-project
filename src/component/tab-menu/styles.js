export const styles = theme => ({
  content: {
    position: "relative",
    minHeight: 256
  }
});
