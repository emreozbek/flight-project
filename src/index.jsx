import React from "react";
import { render } from "react-dom";
import { Provider } from "react-redux";
import { Router } from "react-router";
import App from "./component/App";
import { store, history } from "./store";

render(
  <Provider store={store}>
    <Router history={history}>
      <App />
    </Router>
  </Provider>,
  document.getElementById("app")
);
