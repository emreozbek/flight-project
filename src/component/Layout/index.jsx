import React, { Fragment } from "react";
import PropTypes from "prop-types";
import { Paper, Typography } from "@material-ui/core";
import { withStyles } from "@material-ui/core/styles";
import { styles } from "./styles";

export function Layout(props) {
  const { background, wrapper } = props.classes;
  return (
    <Fragment>
      <div className={background} />
      <div className={wrapper}>
        <Typography variant="h5" component="h1" gutterBottom>
          Flight Project
        </Typography>
        <Paper elevation={1}>{props.children}</Paper>
        <Typography variant="caption">
          Powered using React, Redux, Redux-Saga and Axios
        </Typography>
      </div>
    </Fragment>
  );
}
Layout.propTypes = {
  classes: PropTypes.object.isRequired
};
export default withStyles(styles)(Layout);
