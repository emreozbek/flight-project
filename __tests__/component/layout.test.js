import React from "react";
import renderer from "react-test-renderer";
import { Provider } from "react-redux";
import { withStyles } from "@material-ui/core";
import { store } from "../../src/store";
import { Layout } from "../../src/component/Layout";
import { styles } from "../../src/component/Layout/styles";

const LayoutStyled = withStyles(styles)(Layout);
describe("Layout", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <LayoutStyled />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
