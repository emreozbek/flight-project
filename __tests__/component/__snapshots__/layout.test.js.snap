// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
Array [
  <div
    className="Layout-background-1"
  />,
  <div
    className="Layout-wrapper-2"
  >
    <h1
      className="MuiTypography-root-3 MuiTypography-h5-19 MuiTypography-gutterBottom-30"
    >
      Flight Project
    </h1>
    <div
      className="MuiPaper-root-39 MuiPaper-elevation1-42 MuiPaper-rounded-40"
    />
    <span
      className="MuiTypography-root-3 MuiTypography-caption-13"
    >
      Powered using React, Redux, Redux-Saga and Axios
    </span>
  </div>,
]
`;
